src/components/signVerify/signMessage.js

```diff
@@ -1,4 +1,5 @@
 import lisk from 'lisk-js';
+import './sign-message.less';
```

Comment:
Just for the sake of readability I would do this import also in `src/components/signVerify/verifyMessage.js`
And also rename `sign-message.less` to `signVerifyMessage.less`